### DISCUSSION TITLE
Sync Pop!_OS

### DISCUSSION THREAD

**Discussion by bkpecho** [pop-os/shell#1594] [Q&A] [answered]
Hello everyone. I have two PCs both running [Pop!_OS](https://pop.system76.com/) with VirtualBox. 

I switch using between the two PCs depending on my location. Is there a way that my changes in one instance of pop_os be synced with my other PC? 

Hope you can help a beginner. Thanks!

**Comment by leviport** [comment, marked as answer]
There is an application called Syncthing that might do what you need.

**Reply by bkpecho** [reply]
This is exactly what I needed! Thanks!
